hypervisor/src/cpu.rs

```diff
@@ -495,4 +500,11 @@ pub trait Vcpu: Send + Sync {
     ) -> Result<[u32; 4]> {
         unimplemented!()
     }
+    #[cfg(target_arch = "x86_64")]
+    ///
+    /// Trigger NMI interrupt
+    ///
+    fn nmi(&self) -> Result<()> {
+        Ok(())
```

Comment:
This should be unimplemented by default for hypervisors which do not support this operation yet. Otherwise this would give a wrong impression if nothing happens when someone tries to inject NMI on MSHV based platform.

Reply:
Thanks for pointing this.
Done.